### PR TITLE
Improved air purifier/humidifier related automations

### PR DESCRIPTION
**Pull request by mdrwiega**

### CODE REVIEW

File: automations/air_quality.yaml [hunk 1]
```diff
@@ -72,31 +74,45 @@
...
-      entity_id: sensor.zhimi_airp_mb5_pm2_5
-      above: 20
-  #   - after: '8:00:00'
-  #     before: '23:00:00'
-  #     condition: time
-    # - condition: template
-    #   value_template: "{{ (as_timestamp(now()) - as_timestamp(states.automation.air_quality_notify.attributes.last_triggered | default(0)) | int > 40)}}"
+    and:
+      - condition: time
+        after: '9:00:00'
```
Comment (agnieszkakominek) [inline automations/air_quality.yaml:86]: 8:00:00

File: automations/air_quality.yaml [hunk 2]
```diff
@@ -106,39 +122,22 @@
...
-  #       {% elif state_attr('fan.air_purifier', 'aqi') | int > 20 %}
-  #       4
-  #       {% elif state_attr('fan.air_purifier', 'aqi') | int > 15 %}
-  #       3
-  #       {% else %}
-  #       2
-  #       {% endif %}
+# condition:
+#   conditions:
+#   - after: '1:00:00'
```
Comment (agnieszkakominek) [inline automations/air_quality.yaml:127]: Why this time?

Comment (mdrwiega) [reply automations/air_quality.yaml:127]: not used

File: automations/air_quality.yaml (hunk 1 above)
Comment (mdrwiega) [reply automations/air_quality.yaml:86]: ok

**Review by agnieszkakominek — APPROVED** [review]

**Pull request merged**